docs/configuring-playbook-jitsi.md

```diff
@@ -25,6 +25,17 @@ Add this to your `inventory/host_vars/matrix.DOMAIN/vars.yml` configuration:
 matrix_jitsi_enabled: true
 ```
 
+## Securing your Jitsi instance with strong passwords
+
+Please use the bash script provided in this repo to generate strong passwords for your Jitsi instance.
+Execute the following commands in your terminal from the root of this repo:
+```bash
+cd inventory/scripts
+bash generate-jitsi-passwords.sh
+```
+
+The script will add the corresponding ansible variables and passwords generated with `openssl rand -hex 16` to the bottom of your `inventory/host_vars/matrix.DOMAIN/vars.yml` configuration.
```

Comment:
It would probably be better if the script was just generating passwords and telling you what to put in your `vars.yml` file.

It's less complex and would work fine for people who have more than 1 host in their Matrix inventory.

I guess that if you have more than 1 host, you're likely a more advanced user and can figure out how to generate the passwords manually by yourself, but.. It might be better to be less magical in any case.

Reply:
I've pushed a change (see f798605836e46cc73), which would catch people not defining these settings.

It would also catch people who have previously installed Jitsi with the default secrets (because the defaults are now undefined, so they effectively have no secrets either).

I think what's left to do now is: simplify `generate-jitsi-passwords.sh`, so that it would merely do generation and tell people what to copy/paste and where.

We may also look into simplifying the documentation page a bit.

What are your thoughts on this?

Reply:
I've gone ahead and done this. Looking forward to your feedback though.